### PR TITLE
Am 1036 added column authorisations in ras

### PR DESCRIPTION
**Pull request by dharmendra8186**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-1037

### Change description ###

Adding a new column in RAS - Authorisations

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by ConorH994 — DISMISSED** [review]

File: build.gradle [hunk 1]
```diff
@@ -148,7 +148,7 @@ task functional(type: Test, description: 'Runs the functional tests.', group: 'V
       main = "uk.gov.hmcts.reform.roleassignment.befta.RoleAssignmentBeftaMain"
       classpath += configurations.cucumberRuntime + sourceSets.functionalTest.runtimeClasspath + sourceSets.main.output + sourceSets.test.output
       args = ['--plugin', "json:${projectDir}/target/cucumber.json", '--tags', 'not @Ignore', '--glue',
-              'uk.gov.hmcts.befta.player', 'src/functionalTest/resources/features/']
+              'uk.gov.hmcts.befta.player', 'src/functionalTest/resources/features']
```
Comment (aashishaggarwal123) [inline build.gradle:151]: I think slash is requierd

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -58,8 +58,9 @@ spring:
         jdbc:
           lob:
             non_contextual_creation: true
+    show-sql: true
   liquibase:
-    enabled: ${ENABLE_DB_MIGRATE:false}
+    enabled: ${ENABLE_DB_MIGRATE:true}
```
Comment (aashishaggarwal123) [inline src/main/resources/application.yaml:63]: we need to disable both

File: build.gradle (hunk 1 above)
Comment (dharmendra8186) [reply build.gradle]: done.

File: src/main/resources/application.yaml (hunk 2 above)
Comment (dharmendra8186) [reply src/main/resources/application.yaml]: done.

**Pull request merged**